meta-resin-common/recipes-support/resin-vars/resin-vars.bb

```diff
@@ -96,14 +96,30 @@ do_runtests() {
 	bbnote "Running os-networkmanager tests..."
 	runtest os-networkmanager.testconfig1.json 0 '# This file is generated based on os.networkManager configuration in config.json.
 [device]
-wifi.scan-rand-mac-address=yes'
+wifi.scan-rand-mac-address=yes
+[connectivity]
+uri=/connectivity-check
+interval=3600
+request='
```

Comment:
@ZubairLK from the NM docs you quoted, this should be `response` instead of `request`

Reply:
@pcarranzav nice catch. I think I made a mistake while I was checking on device and editing nm.conf and changing the recipe. Have fixed this